### ISSUE TITLE
Cannot find name 'USVString'.

### ISSUE THREAD

**Issue by quentinus95** [thoov/mock-socket#235]
Hi,

I'm getting a `Cannot find name 'USVString'.` when trying to use this lib using TypeScript.

in

```
 class WebSocket extends EventTarget {
  > 17 |     constructor(url?: USVString, protocols?: string|string[]);
```

My tsconfig.json is:

```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": false,
    "importHelpers": true,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "sourceMap": true,
    "baseUrl": ".",
    "types": [
      "webpack-env",
      "jest"
    ],
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.vue",
    "tests/**/*.ts"
  ],
  "exclude": [
    "node_modules"
  ]
}
```

(generated by Vue CLI)

**Comment by quentinus95** [comment]
Looks like I don't have any bug when using a require instead of an import:

```
const Server = require("mock-socket").Server
```

works

whereas

```
import { Server } from "mock-socket"
```

does not

**Comment by nlaplante** [comment]
Same here, Typescript 3.1.3

**Comment by ekuiter** [comment]
As a workaround, it's possible to override the type definitions by placing the following in `src/node_modules/mock-socket/index.d.ts` (assuming src is the main application source):

```
declare module 'mock-socket' {
    const Server: any;
    //[...]
}
```

See https://stackoverflow.com/a/50794250/1692231.

**Comment by thoov** [comment]
This should be fixed in 8.0.5
